internal/build/options_buildkit.go

```diff
@@ -0,0 +1,26 @@
+// +build buildkit
+
+package build
```

Comment:
waiiiiit how do we differentiate these files?!

Reply:
Check out the build tag in the comment at the top! If the buildkit flag is set, this file will be compiled. The other file has a comment that looks like this:

```golang
// +build !buildkit
```

This means that it will only be compiled if the buildkit flag is _not_ set. They are mutually exclusive.